src/code_1A5830.c

```diff
@@ -392,7 +465,19 @@ ApiStatus GetLastDamage(ScriptInstance* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "code_1A5830", EnableActorGlow);
 
+// Cannot get gBattleStatus to load via lui+addiu no matter what I do
```

Comment:
Probably a .data issue? i.e., gBattleStatus is a global variable defined in this file [it'd make sense, this is the battle engine file]

Reply:
oh! hmm yeah that is possible then!